src/controllers/update/beneficial.owner.type.controller.ts

```diff
@@ -116,6 +127,7 @@ const retrieveManagingOfficers = async (req: Request, appData: ApplicationData)
         } else if (officer.officerRole === "corporate-managing-officer") {
           const managingOfficerCorporate = mapToManagingOfficerCorporate(officer);
           logger.info("Loaded Corporate Managing Officer " + managingOfficerCorporate.id + " is " + managingOfficerCorporate.name);
+          appData.update?.review_managing_officers_corporate?.push(managingOfficerCorporate);
```

Comment:
Do we need this? Reckon it's part of the next review ticket

Reply:
Removed